<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_SendByte</class>
 <widget class="QDialog" name="Form_SendByte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send Byte</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Byte Value</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dec</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Dec_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Hex_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Oct</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Oct_lineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>AscII</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="AscII_tableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>55</number>
        </attribute>
        <row>
         <property name="text">
          <string>00</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>10</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>20</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>30</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>40</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>50</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>60</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>70</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>0</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>5</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>6</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>7</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>8</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>9</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>A</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>B</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>C</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>E</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>F</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>NUL</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>SOH</string>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>STX</string>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>ETX</string>
         </property>
        </item>
        <item row="0" column="4">
         <property name="text">
          <string>EOT</string>
         </property>
        </item>
        <item row="0" column="5">
         <property name="text">
          <string>ENQ</string>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string>ACK</string>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string>BEL</string>
         </property>
        </item>
        <item row="0" column="8">
         <property name="text">
          <string>BS</string>
         </property>
        </item>
        <item row="0" column="9">
         <property name="text">
          <string>TAB</string>
         </property>
        </item>
        <item row="0" column="10">
         <property name="text">
          <string>LF</string>
         </property>
        </item>
        <item row="0" column="11">
         <property name="text">
          <string>VT</string>
         </property>
        </item>
        <item row="0" column="12">
         <property name="text">
          <string>FF</string>
         </property>
        </item>
        <item row="0" column="13">
         <property name="text">
          <string>CR</string>
         </property>
        </item>
        <item row="0" column="14">
         <property name="text">
          <string>SO</string>
         </property>
        </item>
        <item row="0" column="15">
         <property name="text">
          <string>SI</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>DLE</string>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>DC1</string>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>DC2</string>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string>DC3</string>
         </property>
        </item>
        <item row="1" column="4">
         <property name="text">
          <string>DC4</string>
         </property>
        </item>
        <item row="1" column="5">
         <property name="text">
          <string>NAK</string>
         </property>
        </item>
        <item row="1" column="6">
         <property name="text">
          <string>SYN</string>
         </property>
        </item>
        <item row="1" column="7">
         <property name="text">
          <string>ETB</string>
         </property>
        </item>
        <item row="1" column="8">
         <property name="text">
          <string>CAN</string>
         </property>
        </item>
        <item row="1" column="9">
         <property name="text">
          <string>EM</string>
         </property>
        </item>
        <item row="1" column="10">
         <property name="text">
          <string>SUB</string>
         </property>
        </item>
        <item row="1" column="11">
         <property name="text">
          <string>ESC</string>
         </property>
        </item>
        <item row="1" column="12">
         <property name="text">
          <string>FS</string>
         </property>
        </item>
        <item row="1" column="13">
         <property name="text">
          <string>GS</string>
         </property>
        </item>
        <item row="1" column="14">
         <property name="text">
          <string>RS</string>
         </property>
        </item>
        <item row="1" column="15">
         <property name="text">
          <string>US</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>SPC</string>
         </property>
        </item>
        <item row="2" column="1">
         <property name="text">
          <string>!</string>
         </property>
        </item>
        <item row="2" column="2">
         <property name="text">
          <string>&quot;</string>
         </property>
        </item>
        <item row="2" column="3">
         <property name="text">
          <string>#</string>
         </property>
        </item>
        <item row="2" column="4">
         <property name="text">
          <string>$</string>
         </property>
        </item>
        <item row="2" column="5">
         <property name="text">
          <string>%</string>
         </property>
        </item>
        <item row="2" column="6">
         <property name="text">
          <string>&amp;</string>
         </property>
        </item>
        <item row="2" column="7">
         <property name="text">
          <string>'</string>
         </property>
        </item>
        <item row="2" column="8">
         <property name="text">
          <string>(</string>
         </property>
        </item>
        <item row="2" column="9">
         <property name="text">
          <string>)</string>
         </property>
        </item>
        <item row="2" column="10">
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item row="2" column="11">
         <property name="text">
          <string>+</string>
         </property>
        </item>
        <item row="2" column="12">
         <property name="text">
          <string>,</string>
         </property>
        </item>
        <item row="2" column="13">
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item row="2" column="14">
         <property name="text">
          <string>.</string>
         </property>
        </item>
        <item row="2" column="15">
         <property name="text">
          <string>/</string>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="3" column="1">
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item row="3" column="2">
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item row="3" column="3">
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item row="3" column="4">
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item row="3" column="5">
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item row="3" column="6">
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item row="3" column="7">
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item row="3" column="8">
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item row="3" column="9">
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item row="3" column="10">
         <property name="text">
          <string>:</string>
         </property>
        </item>
        <item row="3" column="11">
         <property name="text">
          <string>;</string>
         </property>
        </item>
        <item row="3" column="12">
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item row="3" column="13">
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item row="3" column="14">
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item row="3" column="15">
         <property name="text">
          <string>?</string>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>@</string>
         </property>
        </item>
        <item row="4" column="1">
         <property name="text">
          <string>A</string>
         </property>
        </item>
        <item row="4" column="2">
         <property name="text">
          <string>B</string>
         </property>
        </item>
        <item row="4" column="3">
         <property name="text">
          <string>C</string>
         </property>
        </item>
        <item row="4" column="4">
         <property name="text">
          <string>D</string>
         </property>
        </item>
        <item row="4" column="5">
         <property name="text">
          <string>E</string>
         </property>
        </item>
        <item row="4" column="6">
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item row="4" column="7">
         <property name="text">
          <string>G</string>
         </property>
        </item>
        <item row="4" column="8">
         <property name="text">
          <string>H</string>
         </property>
        </item>
        <item row="4" column="9">
         <property name="text">
          <string>I</string>
         </property>
        </item>
        <item row="4" column="10">
         <property name="text">
          <string>J</string>
         </property>
        </item>
        <item row="4" column="11">
         <property name="text">
          <string>K</string>
         </property>
        </item>
        <item row="4" column="12">
         <property name="text">
          <string>L</string>
         </property>
        </item>
        <item row="4" column="13">
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item row="4" column="14">
         <property name="text">
          <string>N</string>
         </property>
        </item>
        <item row="4" column="15">
         <property name="text">
          <string>O</string>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>P</string>
         </property>
        </item>
        <item row="5" column="1">
         <property name="text">
          <string>Q</string>
         </property>
        </item>
        <item row="5" column="2">
         <property name="text">
          <string>R</string>
         </property>
        </item>
        <item row="5" column="3">
         <property name="text">
          <string>S</string>
         </property>
        </item>
        <item row="5" column="4">
         <property name="text">
          <string>T</string>
         </property>
        </item>
        <item row="5" column="5">
         <property name="text">
          <string>U</string>
         </property>
        </item>
        <item row="5" column="6">
         <property name="text">
          <string>V</string>
         </property>
        </item>
        <item row="5" column="7">
         <property name="text">
          <string>W</string>
         </property>
        </item>
        <item row="5" column="8">
         <property name="text">
          <string>X</string>
         </property>
        </item>
        <item row="5" column="9">
         <property name="text">
          <string>Y</string>
         </property>
        </item>
        <item row="5" column="10">
         <property name="text">
          <string>Z</string>
         </property>
        </item>
        <item row="5" column="11">
         <property name="text">
          <string>[</string>
         </property>
        </item>
        <item row="5" column="12">
         <property name="text">
          <string>\</string>
         </property>
        </item>
        <item row="5" column="13">
         <property name="text">
          <string>]</string>
         </property>
        </item>
        <item row="5" column="14">
         <property name="text">
          <string>^</string>
         </property>
        </item>
        <item row="5" column="15">
         <property name="text">
          <string>_</string>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>`</string>
         </property>
        </item>
        <item row="6" column="1">
         <property name="text">
          <string>a</string>
         </property>
        </item>
        <item row="6" column="2">
         <property name="text">
          <string>b</string>
         </property>
        </item>
        <item row="6" column="3">
         <property name="text">
          <string>c</string>
         </property>
        </item>
        <item row="6" column="4">
         <property name="text">
          <string>d</string>
         </property>
        </item>
        <item row="6" column="5">
         <property name="text">
          <string>e</string>
         </property>
        </item>
        <item row="6" column="6">
         <property name="text">
          <string>f</string>
         </property>
        </item>
        <item row="6" column="7">
         <property name="text">
          <string>g</string>
         </property>
        </item>
        <item row="6" column="8">
         <property name="text">
          <string>h</string>
         </property>
        </item>
        <item row="6" column="9">
         <property name="text">
          <string>i</string>
         </property>
        </item>
        <item row="6" column="10">
         <property name="text">
          <string>j</string>
         </property>
        </item>
        <item row="6" column="11">
         <property name="text">
          <string>k</string>
         </property>
        </item>
        <item row="6" column="12">
         <property name="text">
          <string>l</string>
         </property>
        </item>
        <item row="6" column="13">
         <property name="text">
          <string>m</string>
         </property>
        </item>
        <item row="6" column="14">
         <property name="text">
          <string>n</string>
         </property>
        </item>
        <item row="6" column="15">
         <property name="text">
          <string>o</string>
         </property>
        </item>
        <item row="7" column="0">
         <property name="text">
          <string>p</string>
         </property>
        </item>
        <item row="7" column="1">
         <property name="text">
          <string>q</string>
         </property>
        </item>
        <item row="7" column="2">
         <property name="text">
          <string>r</string>
         </property>
        </item>
        <item row="7" column="3">
         <property name="text">
          <string>s</string>
         </property>
        </item>
        <item row="7" column="4">
         <property name="text">
          <string>t</string>
         </property>
        </item>
        <item row="7" column="5">
         <property name="text">
          <string>u</string>
         </property>
        </item>
        <item row="7" column="6">
         <property name="text">
          <string>v</string>
         </property>
        </item>
        <item row="7" column="7">
         <property name="text">
          <string>w</string>
         </property>
        </item>
        <item row="7" column="8">
         <property name="text">
          <string>x</string>
         </property>
        </item>
        <item row="7" column="9">
         <property name="text">
          <string>y</string>
         </property>
        </item>
        <item row="7" column="10">
         <property name="text">
          <string>z</string>
         </property>
        </item>
        <item row="7" column="11">
         <property name="text">
          <string>{</string>
         </property>
        </item>
        <item row="7" column="12">
         <property name="text">
          <string>|</string>
         </property>
        </item>
        <item row="7" column="13">
         <property name="text">
          <string>}</string>
         </property>
        </item>
        <item row="7" column="14">
         <property name="text">
          <string>~</string>
         </property>
        </item>
        <item row="7" column="15">
         <property name="text">
          <string>DEL</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Form_SendByte</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Form_SendByte</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
